SharkCage/CageConfigurator/CageConfiguratorForm.cs

```diff
@@ -91,6 +94,7 @@ private void CheckPath(string path, string[] file_types, Action<string> onSucces
                 return path.EndsWith(type);
             });
 
+            // fixme: check if path contains cage configurator and disallow it?
```

Comment:
Maybe we should create an issue for that as well?

Reply:
this can just be deleted as we now explicitely want to be able to create a config for the Configurator